stdlib/3/signal.pyi

```diff
@@ -46,6 +48,45 @@ SIGWINCH = ...  # type: int
 SIGXCPU = ...  # type: int
 SIGXFSZ = ...  # type: int
 
+if sys.version_info() >= (3, 5):
```

Comment:
I'm sorry, version_info is not a function, so this should just be `sys.version_info >= (3, 5)`.